tests/rptest/e2e_tests/workloads/flink_table_transactions_basic.py

```diff
@@ -128,6 +128,12 @@ def setup(self):
         table_env = StreamTableEnvironment.create(
             stream_execution_environment=env, environment_settings=settings)
 
+        # Tune table idle state handling
+        # Clear the state if it has not changed
```

Comment:
just curious: are these needed for this fix or these just nice to have

Reply:
This is a safe guard to trigger cleanups/aborts for ongoing actions/transformations. Like we do not want flink to wait infinitely (default) for other party to answer if Table API is querying something in RP